internal/verifier/summary_test.go

```diff
@@ -68,11 +68,11 @@ func TestSummarizeArtifacts(t *testing.T) {
 				}
 			},
 		})
-		canonicalizedHash := hashext.NewMultiHash(crypto.SHA256)
-		canonicalizedZip := must(archivetest.ZipFile([]archive.ZipEntry{
+		stabilizedHash := hashext.NewMultiHash(crypto.SHA256)
+		stableZip := must(archivetest.ZipFile([]archive.ZipEntry{
```

Comment:
nit: Why use "stabilized" as an adjective sometimes vs "stable" other times?

Reply:
here this was due to "stable hash" having a separate meaning while it's less ambiguous than "stable zip."